src/main/java/com/blamejared/controlling/client/gui/GuiNewKeyBindingList.java

```diff
@@ -164,33 +201,67 @@ public void drawEntry(int slotIndex, int x, int y, int listWidth, int slotHeight
             this.btnChangeKeyBinding.yPosition = y;
             this.btnChangeKeyBinding.displayString = GameSettings.getKeyDisplayString(this.keybinding.getKeyCode());
 
-            boolean flag1 = false;
+            boolean hasConflict = false;
 
             if (this.keybinding.getKeyCode() != 0) {
-                for (KeyBinding keybinding : mc.gameSettings.keyBindings) {
-                    if (keybinding != this.keybinding && this.keybinding.getKeyCode() == keybinding.getKeyCode()) {
-                        flag1 = true;
+                for (KeyBinding key : mc.gameSettings.keyBindings) {
+                    if (key != this.keybinding && this.keybinding.getKeyCode() == key.getKeyCode()) {
+                        hasConflict = true;
                         break;
                     }
                 }
             }
 
-            if (flag) {
-                this.btnChangeKeyBinding.displayString = EnumChatFormatting.WHITE + "> "
-                        + EnumChatFormatting.YELLOW
-                        + this.btnChangeKeyBinding.displayString
-                        + EnumChatFormatting.WHITE
-                        + " <";
-            } else if (flag1) {
-                this.btnChangeKeyBinding.displayString = EnumChatFormatting.RED
-                        + this.btnChangeKeyBinding.displayString;
+            final String displayText = GameSettings.getKeyDisplayString(this.keybinding.getKeyCode());
```

Comment:
I suggest to separate method for drawing highlighted string to cleanup code.
Something like:
```
if (highlight) {
    this.drawHightlightedButton(...);
} else {
    this.btnChangeKeyBinding.drawButton(mc, mouseX, mouseY);
}
```